tests/utils/check-tests.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python3
+
+import argparse
+import glob
+import os
+import sys
+import yaml
+import yamlordereddictloader
+
+SUPPORTED_KEYS = [
+    'summary',
+    'details',
+    'backends',
+    'systems',
+    'manual',
+    'priority',
+    'warn-timeout',
+    'kill-timeout',
+    'environment',
+    'prepare',
+    'restore',
+    'debug',
+    'execute'
+    ]
+MANDATORY_KEYS = [
```

Comment:
please run `black` formatter on the file

Reply:
done, thanks for the suggestion